wallet/how-to/get-started-building/run-devnet.md

```diff
@@ -5,24 +5,40 @@ sidebar_position: 2
 
 # Run a development network
 
-You can run a personal Ethereum development network using [Ganache](https://www.trufflesuite.com/ganache),
+You can run a personal Ethereum development network using [Hardhat](https://hardhat.org/),
 which allows you to develop a dapp in a secure test environment.
 
 :::note
-When using a local development blockchain such as Ganache or
+When using a local development blockchain such as Hardhat or
 [anvil](https://book.getfoundry.sh/anvil/#overview-of-anvil), your node must calculate gas to make
 transactions on MetaMask.
 :::
 
-## Connect to Ganache
-
-Follow the [Ganache quickstart](https://trufflesuite.com/docs/ganache/quickstart/) to set
-up a development network.
+## Connect to Hardhat
+
+Follow the [Hardhat Quick Start](https://hardhat.org/hardhat-runner/docs/getting-started#quick-start)
+to set up a Hardhat development network.
+
+In your `hardhat.config.js` file, specify your
+[MetaMask seed phrase](https://support.metamask.io/hc/en-us/articles/360015290032-How-to-reveal-your-Secret-Recovery-Phrase)
+in the [`accounts`](https://hardhat.org/hardhat-network/docs/reference#accounts) field of the
+`networks.hardhat` configuration:
+
+```js title="hardhat.config.js"
+module.exports = {
+  networks: {
+    hardhat: {
+      accounts: {
+        mnemonic: "<your MetaMask seed phrase>",
```

Comment:
safer to save the seed in your env (export SEED_PHRASE="abc 123"), and use process.env.SEED_PHRASE 

This prevents people from accidently adding their seedphrases to a commit. (hopefully they follow the previous step of creating a new seed phrase, but if they end up using that seedphrase at some point, this is a very easy way to leak your key). 

That being said, I can understand why we may want to give this more simple instruction, so feel free to make the call there.